versions/1/data_stream/fields/fields.spec.yml

```diff
@@ -78,9 +78,13 @@ spec:
         type: string
         examples:
           - '^[a-zA-Z]$'
+      source:
+        description: External source
+        type: string
+        enum:
+          - ecs
```

Comment:
So far, we have only "ecs", but in general it might be hard to introduce just a metafile describing dependencies without any code logic. Keep in mind that we need to scrap all field definitions from the ECS repository and process its custom layout.

Long story short - I would be for preserving it hard-coded. We can change it to dynamic later on.

Reply:
Sounds good, yes.